Error: Cannot find module 'Q'

Installing adapt-cli on Debian 8 shows the following error:
    throw err;
          ^
Error: Cannot find module 'Q'

To solve it I had to modify the file:
/usr/local/lib/node_modules/adapt-cli/lib/promise/install.js
and change the 'Q' for 'q' like this:
Q = require('q'),

I think this happens in all case sensitve systems like Linux, Mac OS, etc.

--- a/lib/promise/install.js
+++ b/lib/promise/install.js
@@ -1,5 +1,5 @@
 var bower = require('bower'),
-    Q = require('Q'),
+    Q = require('q'),
     Plugin = require('../Plugin'),
     uninstallPackage = require('./uninstallPackage');
 

